### DISCUSSION TITLE
Melhoria no codigo apresentado no EDITOR HTML

### DISCUSSION THREAD

**Discussion by GustavEvangelista** [balta-io/forum#382] [Tira Dúvidas] [answered]
Gostaria de um feedback sobre esse cod que "fiz". No video o Balta pede como lição de casa que melhorássemos  o cod para desenhar a telinha do editor. E depois de pesquisar acabei encontrando esse aqui. É valido? Eu estudei ele e entendi o que foi feito.

![image](https://github.com/balta-io/forum/assets/141782960/fa48e214-5019-44c8-8a05-29b2fb034322)

**Comment by andrebaltieri** [comment, marked as answer]
@GustaD3V meus parabéns, está correto sim!!! 
Só uma pequena observação, normalmente os nomes de variáveis dentro dos métodos sempre iniciam com minúsculo... mas é algo trivial...

Meus parabéns 🚀

**Reply by GustavEvangelista** [reply]
Obrigado Balta. Daqui a alguns anos, espero estar trabalhando na área, quem sabe na sua equipe! 

Pra cima sempre!
